lib/galaxy/schema/schema.py

```diff
@@ -1902,6 +1902,17 @@ class RoleListModel(BaseModel):
 # Keeping it as a Tuple for now for backward compatibility
 RoleNameIdTuple = Tuple[str, EncodedDatabaseIdField]
 
+# Group_Roles -----------------------------------------------------------------
+
+
+class GroupRoleModel(BaseModel):
+    id: EncodedDatabaseIdField = RoleIdField
+    name: str = RoleNameField
+    url: str = Field(title="URL", description="URL for the group role")
```

Comment:
```suggestion
    url: RelativeUrl = RelativeUrlField
```
The `RelativeUrl` type, for now, is just an alias of `str` but helps to better communicate the type of the field and also allows us to easily replace it just in one place (at some point) with maybe a custom type that validates that it is really a valid relative URL not just any kind of string.